Add initial value selection

T1T4N opened the pull request:
Include newest version of jQuery in folder
Add example page
Add modified jQuery knob script

codemill-id-56e13fc81dc4dd0300fe926d fixes #1

T1T4N committed:
Add initial value selection

Include newest version of jQuery in folder
Add example page

shaharsol commented:
@T1T4N thanks for the PR! two comments:
1. When I run the example, after I click the 1st click (to mark the initialAngel), let's say I click at around 180 degrees, the right side of the knob (0-180 degrees) is getting white. I assume it was meant to mark the point I clicked, but I think there can be a more usable way to mark that, maybe a thin line at about where I clicked?
2. I'm not sure how to query the knob for the initialAngle and the "length of arc" values. can you add to the example a button that upon click alerts the two values so I can understand how to access them from the containing page?

T1T4N committed:
Modify knob display

Add test button showing values

T1T4N commented:
@shaharsol Hello,
I modified the code so the degrees from 0 to the selected initial angle don't get hidden (my logic behind this was to prevent the user from inserting negative angles)
I also added an example button "Show values" that displays our current initial angle and the changed value.
Since there is no direct interface to the knob's internal values, I've added a code that copies the values in our own variables when the 'release' event is triggered.

shaharsol commented:
@T1T4N Thanks Robert.
